docs/dxp/7.x/en/installation-and-upgrades/upgrading-liferay-dxp/other-upgrade-scenarios/upgrading-a-sharded-environment.md

```diff
@@ -1,9 +1,9 @@
 # Upgrading a Sharded Environment
 
 Since Liferay DXP 7.0, Liferay removed its own physical partitioning implementation (also known as sharding) in favor of the capabilities provided natively by database vendors. Upgrading a sharded installation to DXP 7.0 or higher requires migrating it to as many non-sharded Liferay DXP installations (servers) as you have shards. These steps guide you through configuring the new Liferay DXP servers to use your formerly sharded data.
-
+`virtual instances <https://help.liferay.com/hc/en-us/articles/360028818692-Setting-Up-a-Virtual-Instance>`_ 
 ```note::
-   Liferay continues to support its logical partitioning capabilities (also known as [virtual instances](https://help.liferay.com/hc/en-us/articles/360028818692-Setting-Up-a-Virtual-Instance)).
+   Liferay continues to support its logical partitioning capabilities (also known as `virtual instances <https://help.liferay.com/hc/en-us/articles/360028818692-Setting-Up-a-Virtual-Instance>`_ ).
```

Comment:
@jrhoun Can you explain this syntax to me? Why is this link syntax used above the note and here?

...It might be worthwhile to explain it on our Slack channel. :-)

Also, I'm not sure why the

Reply:
the way admonitions are parsed is as RST - so that unfortunately means that when you include formatting in the code braces of an admonition that you need to format it in RST for it to be parsed correctly. Yeah i've been fixing it for most people in small pulls b/c I haven't wanted to force it on people yet....but it will get updated in the doc standards soon and mentioned.